src/schema.ts

```diff
@@ -556,15 +556,66 @@ export const typeDefs = gql`
     relevance: String
   }
 
+  type VisualElementImage {
+    imageUrl: String
+    contentType: String
+  }
+
+  type VisualElementOembed {
+    title: String
+    html: String
+    fullscreen: Boolean
+  }
+
+  type VisualElement {
```

Comment:
Kan denne brukes for emneartikler, eller blir det litt rart.

Reply:
Ser ikke ut som om vi bruker så mye data fra visuelt element andre steder så tror ikke det er nødvendig